master/static/migrations/20231006193809_trigger-by-username--on-users-table.tx.up.sql

```diff
@@ -0,0 +1,6 @@
+DROP TRIGGER IF EXISTS autoupdate_users_modified_at ON users;
+
+CREATE TRIGGER autoupdate_users_modified_at
+  BEFORE INSERT OR UPDATE OF username, password_hash, admin, active, display_name, remote ON public.users
```

Comment:
i believe `id` does not have to be added, right because `id` value wouldn't change?

Reply:
hmm good question I'm not sure but i'd lean toward adding it if there is no db constraint against changing id (I haven't checked the type of it)

Reply:
Id has primary key constraint so i think we dont need to add it for trigger